apps/docs/components/copyButton/CopyButton.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import CopyButton from "./CopyButton";
+
+const meta: Meta<typeof CopyButton> = {
+    title: "Component/Copy/Button",
+    component: CopyButton
+};
+
+export default meta;
+type Story = StoryObj<typeof CopyButton>;
+
+export const Default: Story = {
+    args: {
+        text: "this text can be copied in 1 click",
+        children: "this text can be copied in 1 click"
```

Comment:
I would make text and children different, just so an error doesn't sneak in without noticing